usaspending_api/broker/helpers/last_load_date.py

```diff
@@ -56,13 +56,15 @@ def get_earliest_load_date(keys, default=None):
                 earliest_date = key_date
 
     if earliest_date is None:
-        logger.warning(f"No earliest load date could be calculated because no dates for keys `{keys}` were found!")
+        logger.warning(
+            format_func("No earliest load date could be calculated because no dates for keys `{keys}` were found!")
```

Comment:
Looks like this accidently droped the `f` for format string.
```suggestion
            format_func(f"No earliest load date could be calculated because no dates for keys `{keys}` were found!")
```